### DISCUSSION TITLE
Standard apps should be able to receive browser events

### DISCUSSION THREAD

**Discussion by adamscott** [rxhanson/Multitouch-Community#206] [General]
Apps other than browsers than should support browser events don't seem to react to the input (forward, back).

![Capture d’écran, le 2024-06-07 à 09 39 54](https://github.com/rxhanson/Multitouch-Community/assets/270928/51664d5a-77b5-4294-98ee-5a0d389a0723)

Here, the current setup works on browsers, but doesn't on Visual Studio Code or Obsidian, which are known to go back or forward in "history".

**Comment by rxhanson** [comment]
Multitouch executes keyboard shortcuts for back/forward, and these shortcuts are hard coded for each browser. Allowing this to work for non-browsers would mean either making an assumption for all apps or adding every app I can find. The latter isn't very scalable, so there are a couple options:

1. If all of your apps that you would use back/forward in have the same keyboard shortcut for it, then you could change these gestures to just execute the respective keyboard shortcuts.
2. You can keep your current configuration and add the same gestures again, but configure them to execute the keyboard shortcuts for VSCode & Obsidian, and you could restrict them to those apps if desired.

I don't have plans to change the app specifically for this scenario, unless I come across a better way to handle this situation.
